react_app/src/components/Game/ChallengeCard.js

```diff
@@ -22,12 +22,14 @@ class ChallengeCard extends React.Component {
     this.handleClick = this.handleClick.bind(this);
   }
 
+  // Bad function name :(
   handleClick() {
     this.context.open(this.props.challenge.id);
   }
 
   render() {
-    const challenge = this.props.challenge;
+    // save those money-makers :)
+    const { challenge } = this.props;
```

Comment:
I think I understand what you're doing here (unpacking challenge from props and simultaniously declaring it as a local var) but, "readability counts"...